map2csv/webtool_map2csv.py

```
@@ -0,0 +1,238 @@
+        if click is not None:
+            st.session_state["coords"].append((click["x"], click["y"]))
+            st.rerun()  # クリック直後に即再描画
+        # やり直しボタン
+        if st.button("最後の点を取り消す", key="undo") and st.session_state["coords"]:
+            st.session_state["coords"].pop()
+            if st.session_state["latlons"]:
+                st.session_state["latlons"].pop()
+            st.rerun()
+    with col2:
+        st.write("### 対応する緯度経度を入力（例: 35.6,139.7）")
+        for i, (x, y) in enumerate(st.session_state["coords"]):
```

Ruff (0.11.9)

Loop control variable `x` not used within loop body

(B007)